cmd/eval-dev-quality/cmd/report.go

```diff
@@ -80,6 +81,30 @@ func (command *Report) Execute(args []string) (err error) {
 		command.logger.Panicf("ERROR: %s", err)
 	}
 
+	// Create a CSV file that holds the models meta information.
+	var modelsMetaInformationCSVFile *os.File
+	if modelsMetaInformationCSVFile, err = os.OpenFile(filepath.Join(command.ResultPath, "meta.csv"), os.O_CREATE|os.O_EXCL|os.O_WRONLY, 0755); err != nil {
+		command.logger.Panicf("ERROR: %s", err)
+	}
+	defer modelsMetaInformationCSVFile.Close()
+
+	// Fetch all openrouter models since it is the only provider that currently supports querying meta information.
+	provider := openrouter.NewProvider().(*openrouter.Provider)
+	models, err := provider.Models()
+	if err != nil {
+		command.logger.Panicf("ERROR: %s", err)
+	}
+	var modelsMetaInformation []*model.MetaInformation
+	for _, model := range models {
+		modelsMetaInformation = append(modelsMetaInformation, model.MetaInformation())
+	}
+	metaInformationRecords := report.MetaInformationRecords(modelsMetaInformation)
+	report.SortEvaluationRecords(metaInformationRecords)
```

Comment:
Do the sorting within the function where the records are constructed.

I think you can even use a library:

```go
slices.SortFunc(metaInformationRecords, func(a,b []string){
  return strings.Compare(a[0], b[0])
})
```

Reply:
The `SortEvaluationRecords` function sorts records in a generic way, so I would still use that function

Reply:
Maybe rename it to `SortRecords`